tests/support/server.tcl

```diff
@@ -536,7 +536,7 @@ proc start_server {options {code undefined}} {
     }
 
     # write new configuration to temporary file
-    set config_file [tmpfile redis.conf]
+    set config_file [tmpfile placeholderkv.conf]
```

Comment:
Where possible, I like the idea of removing the name of the project. This makes it easier to refactor later.  
```suggestion
    set config_file [tmpfile server.conf]
```